src/viewport/viewport.service.ts

```diff
@@ -79,19 +26,24 @@ export class ViewportService {
 	/** Observable when viewport size type changes. */
 	sizeType$: Observable<ViewportSizeTypeInfo>;
 
-	private lastWidthCheck: number | undefined;
-	private lastWidthSizeInfo: ViewportSizeTypeInfo | undefined;
+	/** A dictionary of the generated viewport size type info */
+	readonly viewportSizeTypeInfoRefs: Dictionary<ViewportSizeTypeInfo>;
```

Comment:
required by the directive

Reply:
change this to internal only

Reply:
it may be an issue in this case since it is being retrieved through this service from the directive `viewport-matcher.directive`.
perphaps the `viewportSizeTypeInfoList` and `viewportSizeTypeInfoRefs` can be resolved through factory, and then injected into both and used if required

Reply:
yea thats another option, but i meant just do the typedoc `@internal` and maybe also rename prefixed with `_` so `_viewportSizeTypeInfoRefs`

Reply:
Sorry, actually we can expose this as some consumers find it useful.

perhaps rename to `sizeTypeMap`

Reply:
indeed, needed for https://github.com/River-iGaming/odin.ngx/pull/668